e-text-field-type.yml

```diff
@@ -1212,6 +1224,19 @@ components:
         - 'DL class code PW valid from'
```

Comment:
check all DL codes in enum, description should be the same style everywhere as here